README.md

```diff
@@ -43,5 +43,8 @@ Currently, the following generators are available:
 - *generate_python.py*
 - *generate_html.py*
 
+## How to contribute
```

Comment:
The doc is not that big... I would integrate it into the README directly for now.

Reply:
The advantage of a dedicated CONTRIBUTING document is that it is integrated in the github process to create a pull request.

Reply:
Never mind this request then. I overlooked that part in your PR description. Thanks